Assets/MixedRealityToolkit-Examples/UX/Scripts/DialogExamples.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace MixedRealityToolkit.Examples.UX
+{
+    public class DialogExamples : MonoBehaviour
+    {
+        private void Start()
+        {
+            // Optimize the content for immersive headset
+            if (HolographicSettings.IsDisplayOpaque)
+            {
+               GameObject buttonCollection = GameObject.Find("ButtonCollection");
```

Comment:
Where is this button in relation to this GameObject the component will be attached to?

Reply:
The buttons are under ButtonCollection object in the scene hierarchy.
![2018-02-26 17_17_29-unity 2017 2 1f1 personal 64bit - dialogsexample unity - mixedrealitytoolkit-u](https://user-images.githubusercontent.com/13754172/36705290-3063199e-1b19-11e8-9d2c-329c59a9d876.png)

Reply:
Can we get rid of the find method here and just reference it in the inspector? 

The Find should only be a fallback if the inspector value is null.